code/modules/mob/living/emote.dm

```diff
@@ -407,9 +418,17 @@
 	key = "sneeze"
 	key_third_person = "sneezes"
 	message = "sneezes."
+	audio_cooldown = 5
 	message_mime = "acts out an exaggerated silent sneeze."
 	emote_type = EMOTE_VISIBLE | EMOTE_AUDIBLE
 
+/datum/emote/living/sneeze/get_sound(mob/living/user)
+	var/mob/living/carbon/human/human_user = user
```

Comment:
I think I fixed it? I'm not sure I may have misunderstood

Reply:
`gender` is a property which exists on `mob/living`
There is no reason to create the `var/mob/living/carbon/human/human_user` because the only data you want is `gender` which is contained in `unser`.
Additionally casting it to human isn't even appropriate because this emote can be run by things which are not humans.